src/test/scala/edu/ie3/simona/model/grid/SystemComponentSpec.scala

```diff
@@ -62,6 +66,88 @@ class SystemComponentSpec extends UnitSpec {
 
     }
 
+    def setup(): OperationTime.OperationTimeBuilder = {
+      OperationTime.builder()
+    }
+
+    "determine the correct operation interval" in {
+
+      val defaultSimulationStart: ZonedDateTime =
+        TimeUtil.withDefaults.toZonedDateTime("2019-01-01T00:00:00Z")
+      val defaultSimulationEnd: ZonedDateTime =
+        TimeUtil.withDefaults.toZonedDateTime("2019-01-02T00:00:00Z")
```

Comment:
Sorry, I just realized we need to have different times for the test than the default ones. My bad. 

Two options
1. add `with DefaultTestData` to the class defintion above, remove defaultSimulationStart from line 75 / 76 and rename `defaultSimulationEnd` to `simulationEnd`.
2. Only do renaming, so remove the `default` from both variables here.